src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandling.tsx

```diff
@@ -81,6 +99,20 @@ const HenleggBehandling: React.FC<IProps> = ({ onListElementClick, fagsak, behan
             <UIModalWrapper
                 modal={{
                     actions: [
+                        <StyledLenke
```

Comment:
Er ikke dette egentlig en knapp? Den lenker vel ikke noe sted.

Reply:
Ja, enig. Men en knapp skal jo egentlig ikke være stylet som en lenke for å følge UU, og en lenke bør jo ikke oppføre seg som en knapp 🤯  vanskelig å vite hva man skal velge da, men designer foreslo lenke fra designsystem,  så derfor endte det på det. Jeg tror utgangspunktet for lenkevalg her egentlig er at det i fremtiden er tenkt at pdf skal åpnes i ny tab, og da bør det jo være en lenke.. men vet ikke hva som egentlig hadde vært best å putte inn her og nå. Hva tenker du?

Reply:
I tidslinjen for eksempel ble vi enige om at det var greit med et unntak, så der er knapp faktisk stylet som lenke.

Reply:
hmm, skjønner. Jeg tror vi har stylet knapper for dette tidligere, så ville kanskje valgt det nå.

Reply:
... Knapp ble brukt først men designerns motiv for å anbefale lenke (tror jeg) var om stylingen skylle endres ..

Reply:
Dere får avgjøre selv, begge deler går bra :)